content/applications/hr/payroll.rst

```diff
@@ -185,13 +195,13 @@ useful to add may be a `Bonus`.
 To view all the various structures for each structure type, go to :menuselection:`Payroll -->
 Configuration --> Structures`.
```

Comment:
OK thanks- I was just typing what they clicked on, thinking it wasn't needed to add that sub, or that it would be confusing- but I'll remember this for the future